cmd/telemetrygen/internal/logs/logs.go

```diff
@@ -18,6 +19,13 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/cmd/telemetrygen/internal/common"
 )
 
+var (
+	errInvalidTraceIDLenght = fmt.Errorf("TraceID must be a 32 character hex string, like: 'ae87dadd90e9935a4bc9660628efd569'")
+	errInvalidSpanIDLenght  = fmt.Errorf("SpanID must be a 16 character hex string, like: '5828fa4960140870'")
+	errInvalidTraceID       = fmt.Errorf("failed to create traceID byte array from the given traceID, make sure the traceID is a hex representation of a [16]byte, like: 'ae87dadd90e9935a4bc9660628efd569'")
+	errInvalidSpanID        = fmt.Errorf("failed to create SpanID byte array from the given SpanID, make sure the SpanID is a hex representation of a [8]byte, like: '5828fa4960140870'")
```

Comment:
When no format, use `errors.New`

Reply:
I used to think the same, but they are now equivalent and I find it more consistent to use only one for both cases.